### PR TITLE
[NPS] Add NPS' contributor docs

### PR DESCRIPTION
**Pull request by Feranchz**
### What does it do?

Add NPS contributor docs

### CODE REVIEW

**Review by joshuaellis — COMMENTED** [review]
Nbc Brooklyn 99 GIF by Brooklyn Nine-Nine

**Review by markkaylor — COMMENTED** [review]
Nice, just a couple small adjustments :)

**Review by markkaylor — APPROVED** [review]

File: docs/docs/docs/01-core/helper-plugin/hooks/use-persistent-state.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Provides a easily usable hook to store data on the local storage.
+
+## Usage
+
+```js
+import { usePersistentState } from '@strapi/helper-plugin';
+
+const MyComponent = () => {
+  const [navbarOpened, setNavbarOpened] = usePersistentState(true);
```
Comment (gu-stav) [inline docs/docs/docs/01-core/helper-plugin/hooks/use-persistent-state.mdx:17]: I think this example is not fully complete? Should this be:

```suggestion
  const [navbarOpened, setNavbarOpened] = usePersistentState('navbar-open', true);
```

?

**Review by gu-stav — COMMENTED** [review]
Very nice, thank you!

**Review by gu-stav — APPROVED** [review]
💅🏼

**Pull request merged**